.github/workflows/image.yml

```diff
@@ -43,4 +47,10 @@ jobs:
           PASSWORD: ${{ secrets.QUAY_PASSWORD }}
         run: |
           docker login -u $USERNAME -p $PASSWORD quay.io
-          docker push quay.io/hawtio/hawtio-example-quarkus:${{ env.project_version }}
```

Comment:
you removed the example image, but we still use it.

Reply:
it gets pushed in the for loop below with the rest of the images

Reply:
ah, ok. then we dont need the previous step of setting the project version to env, correct? please remove it then.

Reply:
removed the step